test/hotspot/jtreg/runtime/appcds/cacheObject/ArchivedModuleWithCustomImageTest.java

```diff
@@ -25,15 +25,12 @@
  * @test
  * @summary Test archived module graph with custom runtime image
  * @requires vm.cds.archived.java.heap
- * @library /test/jdk/lib/testlibrary /test/lib /test/hotspot/jtreg/runtime/appcds
- * @modules java.base/jdk.internal.module
- *          java.management
- *          jdk.jlink
- *          jdk.compiler
+ * @requires vm.flagless
+ * @library /test/jdk/lib/testlibrary /test/lib /test/hotspot/jtreg/runtime/cds/appcds
```

Comment:
Includes the changes from [JDK-8202339](https://bugs.openjdk.org/browse/JDK-8202339) ( commit [1ed3649](https://github.com/openjdk/jdk/commit/1ed3649f480a0342c99d5db3912f0c69cc856dc4) ) on this file